### ISSUE TITLE
Error creating Query Builder option from Informix Table

### ISSUE THREAD

**Issue by acrey1** [dbeaver/dbeaver#11590]
#### System information: 
- Operating system (distribution) and version: Windows Server 2019
- DBeaver version: 21.0.0 (EE)
- Additional extensions: NA

#### Connection specification:
- Database name and version: Informix 12.x
- Driver name: Informix
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
Error on creating Query Builder window

#### Steps to reproduce, if exist:
On data table, use Ctrl-Click to Open Console.
Console opens with SQL statement in the following form: "SELECT * FROM gamtraining:pronto.deb_master"
Click on Query Builder, or Ctrl-B
Error message that comes up indicates that QB cannot parse the ":" that was created in the SQL statement

#### Include any warning/errors/backtraces from the logs
![image](https://user-images.githubusercontent.com/69837156/109873606-143d6680-7cc2-11eb-82c8-d821379fa415.png)

**Comment by uslss** [comment]
thanks for the report

**Comment by LonwoLonwo** [comment]
Hello @acrey1 
As a workaround, you can remove the name of the DB with the symbol ":" from query ("gamtraining:" in your case), then Visual Query Builder must all correctly parse.

**Comment by LonwoLonwo** [comment]
Waiting for the fix in @JSQLParser

**Comment by LonwoLonwo** [comment]
So, JSQLParser fixed parsing problem with ":", but we chose another way to solve this problem.

https://github.com/dbeaver/dbeaver/issues/6075#issuecomment-808423774 according this comment and those DDL, who returns to us Informix for views, procedures and triggers, we decided to remove the name of the catalog of the generated queries. For Visual Query Builder also.

So, no catalog, no separator - no problem with parsing.
